### PR TITLE
added duplication component to asset list table

### PR DESCRIPTION
**Pull request by bwitkowicz**

### CODE REVIEW

File: src/app/asset-list-table/asset-list-table-action-duplicate/asset-list-table-action-duplicate.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      <ep-asset-list-table #assetTable
+        [assets]="assets"
+        [headerTemplate]="assetListHeaderTemplate"
+        [rowTemplate]="assetListRowTemplate"
+      >
+      </ep-asset-list-table>
+
+      <ng-template #assetListHeaderTemplate>
+        <tr>
+          <th>Duplicate</th>
```
Comment (JerryDoubleU) [inline src/app/asset-list-table/asset-list-table-action-duplicate/asset-list-table-action-duplicate.component.spec.ts]: add translate directive

File: src/app/asset-list-table/asset-list-table-editable-name/asset-list-table-editable-name.scss [hunk 2]
```diff
@@ -12,7 +12,7 @@ ep-asset-list-table {
 
     &:not(:hover) {
       .edit-button {
-        display: none;
+        visibility: hidden;
```
Comment (JerryDoubleU) [inline src/app/asset-list-table/asset-list-table-editable-name/asset-list-table-editable-name.scss]: Why was this changed?

Comment (bwitkowicz) [reply src/app/asset-list-table/asset-list-table-editable-name/asset-list-table-editable-name.scss]: fixed that
